cray_product_catalog/schema/schema.yaml

```diff
@@ -147,4 +151,26 @@ definitions:
         description: The type of repository
         type: string
         enum:
-          - group
\ No newline at end of file
+          - group
+  S3Repositories:
```

Comment:
For these new schemas being added, do we ever want to allow empty string values for the manifests, buckets, or keys? It seems like we would not want that. In that case, consider adding `minLength: 1` for those items (and adding tests for it).